Sources/Concrete/ViewModel/MultiHitsViewModel.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  MultiHitsViewModel.swift
+//  InstantSearch-iOS
+//
+//  Created by Guy Daher on 24/11/2017.
+//
+
+import Foundation
+import InstantSearchCore
+
+/// ViewModel - View: HitsViewModelDelegate.
+///
+/// ViewModel - Searcher: SearchableViewModel, ResultingDelegate, ResettableDelegate.
+public class MultiHitsViewModel: MultiHitsViewModelDelegate, SearchableMultiIndexViewModel {
+    
+    // MARK: - Properties
+    
+    public var indexNamesArray: [String] {
```

Comment:
Why do we need to separate arrays for names and ids instead of a struct having both ?

Reply:
same comment as before